[travis] C building: weird behavior at travis server, -lm not found

```
$ python stats.py --all --build

Command: All Build
Builded Problem001/C/solution_1.c: Answer: 233168: 0.00s
Builded Problem002/C/solution_1.c: Answer: 4613732: 0.00s
@Building next Problem027/C/solution_1.c:  /tmp/cch6tzam.o: In function `is_prime':

/home/travis/build/DestructHub/ProjectEuler/Problem027/C/solution_1.c:36: undefined reference to `sqrt'

collect2: ld returned 1 exit status
```

```ranu@ranu-laptop ~/Github/DestructHub/ProjectEuler/Problem048/C (makefile-fix) $ gcc -lm -Wall solution_1.c
/tmp/cczRZ3c1.o: In function `main':
solution_1.c:(.text+0xa3): undefined reference to `pow'
solution_1.c:(.text+0xbc): undefined reference to `fmod'
collect2: error: ld returned 1 exit status
```

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -16,5 +16,6 @@ services:
                               --s c++
                               --s bash
                               --s clojure
+                              --s c
     volumes:
       - .:/code

--- a/stats.py
+++ b/stats.py
@@ -73,7 +73,7 @@ class Build(Checker):
     fout = "compiled.out"
 
     def compile(self):
-        args = self.compiler + [self.path, "-o", self.output]
+        args = [self.compiler[0], self.path, "-o", self.output] + self.compiler[1:]
         program = subprocess.Popen(args, stdout=subprocess.PIPE)
         return program.wait() == 0
 

